src/main/resources/rhsm-subscriptions.properties

```diff
@@ -5,6 +5,12 @@
 # We're keeping the context-path as "/" so that the actuators we use for OKD liveness/readiness probes have a
 # fixed path.  The actual subscriptions resources will be configured to hang off of the path below.
 
+# General hibernate configurations
+spring.jpa.properties.hibernate.jdbc.batch_size=${JDBC_BATCH_SIZE:100}
+spring.jpa.properties.hibernate.order_inserts=true
+spring.jpa.properties.hibernate.order_updates=true
+#spring.jpa.properties.hibernate.batch_versioned_data=true
```

Comment:
Can you please leave a comment as to why we would uncomment this, and what default value is (looks like default is `false`?)

Reply:
Actually, that wasn't supposed to get in there. That was an oversight on my part and would only be used if we were versioning our entities.

I'll remove it.